src/domains/fdr-app/env/weu-dev/terraform.tfvars

```diff
@@ -129,4 +129,8 @@ fdr_json_to_xml_function_autoscale = {
 
 ftp_organization = "55555555555,90000000002,88888888888,11111111111,paStress4,44444444444,19721972197,11111122222,66666666666,55555666666,89898989898,20000000002,11111122223,11223344551,15376371009_FTP"
 
-enable_fdr3_features = true
+enable_fdr3_features           = true
+enable_fdr_org_soap_request    = true
+enable_fdr_psp_soap_request    = true
+fdr_soap_request_psp_whitelist = "*"
```

Comment:
Can you provide value for `fdr_soap_request_psp_whitelist` and `fdr_soap_request_ci_whitelist` for UAT and PROD environment, no matter if fake or not? IIRC, the named values always requires a value to be set.

Reply:
For example in UAT
ci: `88888888888,15376371009`
psp: `88888888888`

in these cases only the fdr flows from that PSP to these CI will be redirect to our service.